.github/workflows/Regression.yml

```
@@ -211,6 +211,50 @@ jobs:
            echo "Storage bump detected, all good!"
         fi
 
+ regression-test-binary-size:
+  name: Regression test binary size
+  runs-on: ubuntu-20.04
+  env:
+    CC: gcc-10
+    CXX: g++-10
+    GEN: ninja
+    BUILD_TPCH: 1
+    BUILD_TPCDS: 1
+    BUILD_JSON: 1
+    BUILD_PARQUET: 1
```

```suggestion
    BUILD_PARQUET: 1
    EXTENSION_STATIC_BUILD: 1
```
I think this is needed to actually show differences in a model that is closer to regular deployment of extensions